regular_expressions_intermediate/regular_expressions_intermediate.md

```diff
@@ -0,0 +1,540 @@
+<!--
+
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+current_version_description: Initial version
+module_type: standard
+docs_version: 1.0.0
+language: en
+narrator: US English Female
+mode: Textbook
+previous_sequential_module: regular_expressions_basics
+
+title: Regular Expressions: Intermediate Level
```

Comment:
suggestion: make this title match the format of Regular Expressions Basics (or make that title match this format)

Reply:
@drelliche I was trying to mimic the SQL series here, which has Basics and Intermediate, but I'm game to change things up!  Below I put in a comma instead of a colon, does this help?
```suggestion
title: Regular Expressions, Intermediate Level
```